W-14139140 added doc about deploy and redeploy flags (#240)
mulesoft/docs-anypoint-cli: modules/ROOT/pages/_partials/api-mgr.adoc

| The CloudHub Gateway version
| `--gatewayVersion: 9.9.9.9  643404`

| `--javaVersion`
| Gateway Java version
| `--javaVersion 17 643404`

| `--releaseChannel`
| Release channel

> | `--releaseChannel EDGE 643404`

Reviewer: Can you specify for which deployment types is it used?

Author: sure!

| The CloudHub Gateway version
| `--gatewayVersion: 9.9.9.9  643404`

| `--javaVersion`
| Gateway Java version +
This flag is used by `--target` `RTF`, `CH`, and `CH2`
| `--javaVersion 17 643404`

| `--releaseChannel`
| Set the name of the release channel to be used for the selected Mule version. + Supported values are `NONE`, `EDGE`, and `LTS` +
This flag is used by `--target` `RTF`, `CH`, and `CH2`
| `--releaseChannel EDGE 643404`

| `--overwrite`

> | Update application if exists +

Reviewer 2: ```suggestion
| Update application if it exists +
```

Include the flag to enable it | `--overwrite`

| `--target`

> | Hybrid, RTF, CH or CH2 deployment target ID

Reviewer 2: ```suggestion
| Hybrid, RTF, CH, or CH2 deployment target ID
```

| `--gatewayVersion`
| CloudHub Gateway version | `--gatewayVersion 9.9.9 643404`

| `--javaVersion`
| Gateway Java version
| `--javaVersion 17 643404`

| `--releaseChannel`
| Release channel

> | `--releaseChannel EDGE 643404`

Reviewer: Same

| `--gatewayVersion`
| CloudHub Gateway version | `--gatewayVersion 9.9.9 643404`

| `--javaVersion`
| Gateway Java version +

> This flag is used by `--target` `RTF`, `CH`, and `CH2`

Reviewer 2: ```suggestion
This flag only works if the target flag is set as `RTF`, `CH`, or `CH2`

| `--javaVersion 17 643404`

| `--releaseChannel`
| Set the name of the release channel to be used for the selected Mule version. + Supported values are `NONE`, `EDGE`, and `LTS` +

> This flag is used by `--target` `RTF`, `CH`, and `CH2`

Reviewer 2: ```suggestion
This flag only works if the target flag is set as `RTF`, `CH`, or `CH2` 
```

| `--releaseChannel EDGE 643404`

| `--overwrite`
| Update application if exists +
Include the flag to enable it | `--overwrite`

| `--target`

> | Hybrid, RTF, CH or CH2 deployment target ID 

Reviewer 2: ```suggestion
| Hybrid, RTF, CH, or CH2 deployment target ID 
```
